test/example.test.js

```diff
@@ -1,18 +1,37 @@
 // IMPORT MODULES under test here:
 // import { example } from '../example.js';
+import { renderDetails } from '../render-utils.js';
 
 const test = QUnit.test;
 
-test('time to test a function', (expect) => {
+test('renderFamily should return <li> with family info', (expect) => {
     //Arrange
     // Set up your arguments and expectations
-    const expected = true;
-    
-    //Act 
-    // Call the function you're testing and set the result to a const
-    const actual = true;
+    const expected = `<li class="member-list"><a href="/details/?id=Amanda"></a></li>`;
 
+    //Act
+    // Call the function you're testing and set the result to a const
+    const actual = renderDetails({
```

Comment:
You'll want to make sure the test string on line 7 and the function you are testing matches to avoid confusion